packages/material-ui-lab/src/TreeView/TreeView.d.ts

```diff
@@ -37,6 +37,18 @@ export interface TreeViewPropsBase
    * Expanded node ids. (Controlled)
    */
   expanded?: string[];
+  /**
+   * This prop is used to help implement the accessibility logic.
+   * If you don't provide this prop. It falls back to a randomly generated id.
+   */
+  id?: string;
+  /**
+   * Callback fired when tree items are focused.
+   *
+   * @param {object} event The event source of the callback
+   * @param {string} value of the focused node.
+   */
+  onNodeFocus?: (event: React.ChangeEvent<{}>, nodeId: string) => void;
```

Comment:
Since you can no longer use onFocus to track focus to nodes thought I should add this.